GiEnJul/ClientApp/src/components/admin/common/RecipientTable.tsx

```diff
@@ -37,14 +44,6 @@ const DatatableRecipient: React.FC<Props> = ({ data }) => {
     }
   };
 
-  const handleMatched = (input: Boolean) => {
-    if (input) {
-      return classes.matched;
-    } else {
-      return classes.notMatched;
-    }
-  };
-
   const handleGender = (gender: Number, age: Number) => {
```

Comment:
There is a gender enum in `common/enums/Gender.tsx` that can be used instead of numbers. This will improve readability. 

I think the mapping to string should be done using a common function, if possible. There is one in `common/constants/Genders.tsx` that has an association between the Gender enum and strings. 
This doesn't map based on age though. I don't think this is necessary though, as this is only for admins. The `constants/Genders.tsx` could maybe be converted into a function that takes Gender and age as arguments, although I think this won't be needed.

Reply:
I think we should include the difference between gutt/mann even though is for admin. We will have to do it this way in the email we send to the givers so why not keep it consistent.

Reply:
That's true. Would it be better to change the `/constants/Genders.tsx` into a function then that uses gender and age as arguments? If age is undefined "mann"/"kvinne" could be returned. I think this would be nice for clarity and reusability.
This may be a rather big task though and could be put into its own issue.

Reply:
Gender is implemented. We can create an issue for creating a functions since we have to change other parts of the code when doing this, such as institutionForm.